app/src/main/java/br/com/connectattoo/repository/ProfileRepository.kt

```diff
@@ -53,4 +55,33 @@ class ProfileRepository(private val tattooClientProfileDao: TattooClientProfileD
         }
 
     }
+
+    fun deleteProfilePhoto(token: String): Flow<ResourceResult<String>> = flow {
+        emit(deleteProfilePhotoAPIAndRoom("Bearer $token"))
+    }
+    private suspend fun deleteProfilePhotoAPIAndRoom(token: String): ResourceResult<String> {
+        try {
+            with(apiService.deleteProfilePhoto(token)) {
+                return resourceResultDeleteProfilePhoto()
+            }
+
+        } catch (error: IOException) {
+            val message = MessageException("Erro na requisição a api")
+            Log.e(TAG, error.message.toString())
+            return (ResourceResult.Error(null, message))
+        }
+
+    }
+
+    private suspend fun Response<String>.resourceResultDeleteProfilePhoto(): ResourceResult<String> {
+        val message = this.code()
+        return if (message == CODE_SUCCESS_204) {
```

Comment:
Isso não deveria funcionar.

Reply:
No caso, se for deletado e retornar o código 200 entraria aqui. this.code() retorna o código de resposta da API.

Reply:
Temos, verifica se no teu Postman ta desatualizado
https://www.postman.com/connectattoo-api/workspace/api-development/request/31592849-5e04b0b3-cd67-490d-94ba-357169582994?action=share&source=copy-link&creator=31592849&ctx=documentation

Reply:
Entendi. Eu falo a respeito dos testes, esse código foi testado pra verificar se o fluxo funciona como deveria?
Pois se fosse testado deveria falhar, já que o código de status seria 200

Reply:
Blz, foi alterado para código 200 e testado.